Project.toml

```diff
@@ -25,6 +25,7 @@ p7zip_jll = "3f19e933-33d8-53b3-aaab-bd5110c3b7a0"
 
 [extras]
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Preferences = "21216c6a-2e73-6563-6e65-726566657250"
 
 [targets]
-test = ["Test"]
+test = ["Test", "Preferences"]
```

Comment:
I'm not sure how the Julia CI tests will work with an stdlib depending on non-stdlib packages.

Reply:
Could `Pkg.add` it from within `runtests.jl` instead

Reply:
Indeed a problem:

```
  Got exception outside of a @test
  LoadError: ArgumentError: Package Preferences not found in current path.
  - Run `import Pkg; Pkg.add("Preferences")` to install the Preferences package.
  Stacktrace:
```

with:
```
JULIA_DEPOT_PATH=(mktemp -d) make -C test Pkg
```